spec/lib/ak_donation_action_creator_spec.rb

```diff
@@ -25,8 +25,10 @@
   }
 
   it 'calls the endpoint for creating a donation action in ActionKit' do
-    stub_request(:post, 'https://fake_username:fake_password@act.sumofus.org/rest/v1/action/')
-    subject.create_donation_action(full_donation_options)
-    expect(WebMock).to have_requested(:post, 'https://fake_username:fake_password@act.sumofus.org/rest/v1/donationpush/')
+    VCR.use_cassette('AkDonationActionCreator') do
+      stub_request(:post, 'https://fake_username:fake_password@act.sumofus.org/rest/v1/action/')
+      subject.create_donation_action(full_donation_options)
+      expect(WebMock).to have_requested(:post, 'https://fake_username:fake_password@act.sumofus.org/rest/v1/donationpush/')
```

Comment:
So basically this isn't a spec I wrote - I didn't really have a look at it. I disabled making external requests if a VCR cassette didn't exist, and this spec started failing, so I added a cassette to it as a quick fix. I don't know what the AkDonationActionCreator does and it wasn't something that was touched by my PR, so I didn't fix the spec.